package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Tracks deployments of sample applications",
   "main": "app.js",
   "scripts": {
-    "postinstall": "./admin.js track && ./admin.js db put && ./admin.js ddoc put && ./admin.js clean repository_url_hash",
+    "postinstall": "bower install --allow-root && ./admin.js track && ./admin.js db put && ./admin.js ddoc put && ./admin.js clean repository_url_hash",
```

Comment:
When bower runs in Bluemix it actually runs as root, bower throws warnings all over the place if it runs as root.